ToolBox/IQFeed/IQFeedFileHistoryProvider.cs

```diff
@@ -270,6 +270,28 @@ private static string GetHistoryRequestKey(string ticker, DateTime startDate, Da
             return $"{ticker}-{startDate}-{endDate}";
         }
 
+        /// <summary>
+        /// Prevent IQFeed from returning unordered data of all sort. This might happen exceptionally.
+        /// </summary>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        private static IEnumerable<BaseData> FilterUnorderedData(IEnumerable<BaseData> data)
```

Comment:
Curious why we filter instead of ordering the data?

Reply:
good question... basically its just unexpected but might happen. I dont what to impact the overperformance for 0.001%

Reply:
Sounds good 👍